pixart.js

```diff
@@ -1 +1,36 @@
+var button = document.querySelector("#set-color");
+var colorText = document.querySelector("#color-field");
+var bigSquare = document.querySelector(".brush");
+var color = "#1B4370";
+var pixels = document.querySelector(".pixels");
 
+playPixart();
+
+function playPixart() {
+  button.addEventListener("click", setColor); //This also works when "enter" is pressed on the keyboard. Why? Should I create a separate event to ensure this event happens for keying enter on any computer?
```

Comment:
This is picky, but in trying to make your code more modular, I might even just call `addSquare()` below `playPixart() `on line 8. It's totally fine to do it here, but I think that this particular function should only be in charge of doing one thing-setting up event listeners. Just a thought :)